test/fixtures/fixture-sharepointer.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function(){
```

Comment:
No worries. Recommendations around sinon are built on my limited usage of late, so may not be completely practical in all cases.
There's always a call on whether to stub functions as needed or mock out an entire module. Seeing the same thing in java land on when to use Mockito vs. a completely 'test' implementation of a class.